diff --git a/node_modules/whatwg-url/dist/encoding.js b/node_modules/whatwg-url/dist/encoding.js
index cb66b8f..4365554 100644
--- a/node_modules/whatwg-url/dist/encoding.js
+++ b/node_modules/whatwg-url/dist/encoding.js
@@ -1,6 +1,7 @@
 "use strict";
-const utf8Encoder = new TextEncoder();
-const utf8Decoder = new TextDecoder("utf-8", { ignoreBOM: true });
+var util = require('util');
+const utf8Encoder = new util.TextEncoder();
+const utf8Decoder = new util.TextDecoder("utf-8", { ignoreBOM: true });
 
 function utf8Encode(string) {
   return utf8Encoder.encode(string);
